__tests__/app.test.js

```diff
@@ -84,4 +84,78 @@ describe("nc_games_test", () => {
         });
     });
   });
+  describe("/api/reviews", () => {
+    it("200: GET /api/reviews should return array of objects.", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          const { reviews } = body;
+          expect(reviews).toBeInstanceOf(Array);
+          reviews.forEach((review) => {
+            expect(review).toBeInstanceOf(Object);
+          });
+        });
+    });
+    it("200: GET /api/reviews should have expected shape with added column for comment count", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          const { reviews } = body;
+          expect(reviews.length).toBe(13);
+          reviews.forEach((review) => {
+            expect(review).toHaveProperty("review_id", expect.any(Number)),
+              expect(review).toHaveProperty("title", expect.any(String)),
+              expect(review).toHaveProperty("review_body", expect.any(String)),
+              expect(review).toHaveProperty("designer", expect.any(String)),
+              expect(review).toHaveProperty(
+                "review_img_url",
+                expect.any(String)
+              );
+            "https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?w=700&h=700",
+              expect(review).toHaveProperty("votes", expect.any(Number)),
+              expect(review).toHaveProperty("category", expect.any(String)),
+              expect(review).toHaveProperty("owner", expect.any(String));
+            expect(review).toHaveProperty("comment_count", expect.any(String));
+          });
+        });
+    });
+    it("404: Should respond with invalid path message if given invalid path input", () => {
```

Comment:
Technically this error isn't associated with this endpoint so should be extracted out to its own describe block. We will only need one test like this as it cover all bad paths.